src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1494,6 +1494,12 @@ internal static void ApplyConfiguration(JObject config, ScriptHostConfiguration
                 }
             }
 
+            JToken nugetFallbackFolder = config["nugetFallbackFolder"];
+            if (nugetFallbackFolder != null && nugetFallbackFolder.Type == JTokenType.String)
```

Comment:
Tests missing?

Reply:
for some reason fallback folder is not taking effect. will add tests as soon i find the root cause.

Reply:
please open an issue to add tests